tox.ini

```diff
@@ -1,20 +1,21 @@
 [tox]
 minversion = 3.9.0
-envlist = py39, py310, py311, flake8, mypy
+envlist = {py39, py310, py311}-{standard, xdist}, flake8, mypy
```

Comment:
Running with and without pytest-xdist seems unnecessary here IMO, since the other tests won't invoke it. Just set this to run once with xdist installed.

Reply:
I think it would be interesting to see if adding `pytest-xdist` would break certain features.

Reply:
I've changed my mind. I removed the matrix.